packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -2310,6 +2311,15 @@ export function enqueuePendingPassiveHookEffectUnmount(
 ): void {
   if (runAllPassiveEffectDestroysBeforeCreates) {
     pendingPassiveHookEffectsUnmount.push(effect, fiber);
+    if (__DEV__) {
+      if (deferPassiveEffectCleanupDuringUnmount) {
+        fiber.effectTag |= PassiveUnmountPending;
```

Comment:
Is it weird to only set this bit in DEV mode? It's only needed there.

Reply:
Add Dev suffic to the flag?